### PR TITLE
change "use" and "Group" in register.go files

### PR DESCRIPTION
**Pull request by liangwenguo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
We have such wording in almost all of our api groups. I'm not sure if this change is correct and we want to change all the other places. @deads2k @liggitt @smarterclayton WDYT?

**Review by dixudx — CHANGES_REQUESTED** [review]
Please also address other `register.go` files.

**Comment by liangwenguo** [comment]
@xiangpengzhao @dixudx @freeformz PTAL, Thank you!

File: staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func Kind(kind string) schema.GroupKind {
 	return SchemeGroupVersion.WithKind(kind).GroupKind()
 }
```
Comment (dixudx) [inline staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: Please also remove `Group` in L30.

Address this in other files as well.

Comment (liangwenguo) [reply staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: Please take a look again, , Thank you!

**Comment by liangwenguo** [comment]
@dixudx Please take a look again, Thank you!

**Comment by dixudx** [comment]
@liangwenguo  Please change the PR title to be more concise. Thanks.

/lgtm

**Comment by dixudx** [comment]
@kubernetes/sig-api-machinery-api-reviews PTAL.

**Review by timothysc — APPROVED** [review]
/approve

**Comment by liangwenguo** [comment]
The local branch has been deleted and cannot be modified, close it

**Pull request closed without merging**